src/QueryBuilder/Condition/Builder/BetweenBuilder.php

```diff
@@ -38,7 +38,7 @@ public function __construct(private readonly QueryBuilderInterface $queryBuilder
      */
     public function build(ExpressionInterface $expression, array &$params = []): string
     {
-        $operator = $expression->operator;
+        $operator = $expression->isNot() ? 'NOT BETWEEN' : 'BETWEEN';
```

Comment:
Get operator from class name like for `AndX` and `OrX`
```php
$operator = match ($expression::class) {
    Between::class => 'BETWEEN',
    NotBetween::class => 'NOT BETWEEN',
};
```

Reply:
Done.